sparse-14/etc/dconf/db/vendor.d/lipstick-configs.txt

```diff
@@ -0,0 +1,3 @@
+[desktop/lipstick-jolla-home/peekfilter]
+boundaryWidth=48
```

Comment:
This is ought to be too small.

Values from Xperia 10 III:
```
boundaryHeight=60
boundaryWidth=60
keyboardBoundaryHeight=30
keyboardBoundaryWidth=30
pressDelay=800
```

I'd go with these.